bin/templates/cordova/lib/check_reqs.js

```diff
@@ -21,6 +21,7 @@
 
 var shelljs = require('shelljs');
 var child_process = require('child_process');
+var Q = require('q');
```

Comment:
This was added back?

Reply:
No mistake. Those are the instances removed by the execa PR. The idea was to merge this into your Q PR, then merge the execa PR, rebase this and merge this. Then all of A should be gone without us having to create any intermediary workarounds for the spawn stuff